<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenamePreset</class>
 <widget class="QWidget" name="RenamePreset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RenamePreset</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Preset Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Thin Medium</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveNameButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DiscardNameButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>DISCARD</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
